cmd/gloat/main.go

```diff
@@ -97,6 +110,94 @@ func upCmd(args arguments) error {
 	return nil
 }
 
+func latestCmd(args arguments) error {
+	gl, err := setupGloat(args)
+	if err != nil {
+		return err
+	}
+
+	latest, err := gl.Latest()
+	if err != nil {
+		return err
+	}
+
+	if latest != nil {
+		fmt.Printf("%d", latest.Version)
+	}
+	return nil
+}
+
+func presentCmd(args arguments) error {
+	gl, err := setupGloat(args)
+	if err != nil {
+		return err
+	}
+
+	migrations, err := gl.Present()
+	if err != nil {
+		return err
+	}
+
+	for i, m := range migrations {
+		fmt.Printf("%d", m.Version)
+		if i != len(migrations)-1 {
+			fmt.Print(",")
+		}
+	}
+
+	return nil
+}
+
+func currentCmd(args arguments) error {
+	gl, err := setupGloat(args)
+	if err != nil {
+		return err
+	}
+
+	current, err := gl.Current()
+	if err != nil {
+		return err
+	}
+
+	if current != nil {
+		fmt.Printf("%d", current.Version)
+	}
+	return nil
+}
+
+func toCmd(args arguments) error {
```

Comment:
By convention the name `toCmd` looks like it should convert the arguments to a `Cmd`, can we improve the naming a bit?